docs/docker.md

```diff
@@ -15,4 +15,4 @@ More background information and optional hints:
 - To run the containers in production the `runc` command is used which does not use the `ENTRYPOINT` specified in the `Dockerfile`.
   It would still be good to have an `ENTRYPOINT` specified for local use and testing.
 - The students solution is currently mounted to `/mnt/exercism-iteration/` in production.
-  We still pass in the folder as a second argument to the analyzer in case that changes in the future.
+  We still pass in the solution folder as the second argument to the analyzer in case that changes in the future.
```

Comment:
You could change this now too?

Reply:
I'll rephrase a bit

Reply:
Do we even need this bit of text?

Reply:
If we don't mention this, maintainers also can't rely on it.

Reply:
Yeah - get rid of it.